src/docs/signals/anomalies/index.md

```diff
@@ -60,7 +60,7 @@ List of supported tokens:
 * omisego
 * santiment
 
-List of supported metrics:
+### List of supported metrics:
```

Comment:
offtopic: This list is no longer correct - only circulation 1d is supported of all circulations.